test/test_transforms.py

```diff
@@ -1685,6 +1685,67 @@ def test_random_erasing(self):
             img_re = random_erasing(img)
             self.assertTrue(torch.equal(img_re, img))
 
+    def test_compose_non_scriptable(self):
+        # T.Compose is now nn.Module, we should check BC with any exotic ops inside
+        # like Lambda, custom function etc
+        img_np = np.random.randint(0, 256, (34, 42, 3), np.uint8)
+        img_pil = Image.fromarray(img_np, mode='RGB')
+
+        def custom_transform(img):
+            return Image.blend(img, img, alpha=0.5)
+
+        t = transforms.Compose([
+            transforms.RandomResizedCrop(15),
+            transforms.Lambda(lambda x: x),
+            custom_transform,
+        ])
+
+        t.transforms.append(transforms.ToTensor())
+        t.transforms.append(lambda x: x + 2)  # THIS DOES NOT WORK
```

Comment:
Can you explain what this doesn't work?

Reply:
This fails due to the fact that `lambda x: x + 2` is not `nn.Module`. Here we modify constructed Compose with custom ops that are not wrapped by Lambda.
See also the discussion [above](https://github.com/pytorch/vision/pull/2645#discussion_r487965999).